bumpversion/cli.py

```diff
@@ -103,6 +103,10 @@ def main(original_args=None):
     args, file_names = _parse_arguments_phase_3(remaining_argv, positionals, defaults, parser2)
     new_version = _parse_new_version(args, new_version, version_config)
 
+    # do not use the files, from the config
+    if args.no_configured_files:
+        files = []
```

Comment:
I think it would be clearer to dot his right after calling `_load_configuration`.

Reply:
this is not possible because args is at that point a list - it probably gets populated later with the commandline arguments